src/assets/scripts/airport/StandardRoute/StandardRouteModel.js

```diff
@@ -363,6 +363,11 @@ export default class StandardRouteModel extends BaseModel {
         } else if (_has(standardRoute, 'exitPoints')) {
             this._entryCollection = this._buildSegmentCollection(standardRoute.rwy);
             this._exitCollection = this._buildSegmentCollection(standardRoute.exitPoints);
+        } else if (_has(standardRoute, 'rwy')) {
+            console.error(`${this.icao} - (${this.name}) Does not contain exitPoints or entryPoints. ` +
+                `Please update the '${this.icao.toLowerCase()}.json' file to include exitPoints, if this is a SID, ` +
+                `or entryPoints if this is a STAR.`);
```

Comment:
@n8rzz Actually, you're right, we do need a change. The `'${this.icao.toLowerCase()}.json'` does not work, because you can't get the airport's ICAO identifier from the `StandardRoute`; what you end up getting here is the identifier of the procedure, for which there is no file. Recommendation:

```javascript
console.error(`The '${this.icao}' procedure does not contain exitPoints or entryPoints. ` +
    `If this is a SID, at least one exitPoint must be defined. If this is a STAR, at least ` +
    `one entryPoint must be defined.`);
```